data/resources/ui/sidebar-row.ui

```diff
@@ -98,28 +98,41 @@
               </object>
             </child>
             <child>
-              <object class="GtkImage" id="pin_icon">
-                <property name="icon-name">view-pin-symbolic</property>
-                <style>
-                  <class name="dim-label"/>
-                </style>
-              </object>
-            </child>
-            <child>
-              <object class="GtkLabel" id="unread_mention_label">
-                <property name="valign">center</property>
-                <property name="justify">center</property>
-                <property name="label">@</property>
-                <style>
-                  <class name="unread-mention-count"/>
-                </style>
-              </object>
-            </child>
-            <child>
-              <object class="GtkLabel" id="unread_count_label">
-                <property name="valign">center</property>
-                <property name="ellipsize">end</property>
-                <property name="justify">center</property>
+              <object class="GtkStack" id="status_stack">
+                <child>
+                  <object class="AdwBin" id="empty_status_bin"/>
```

Comment:
Hmm. I'm not sure. That would force us to use two expressions, making the code cumbersome and a bit bloated.
The `empty-bin` makes the intention clear that a chat can have a `None` state.

Nevertheless, I'm fine to do it the way you suggested. So, after expressing my concerns I pushed the last commit a demonstration how it would look like making it invisible (needs still some work).

Reply:
> Hmm. I'm not sure. That would force us to use two expressions, making the code cumbersome and a bit bloated.

Ok, fair, let's go for the `empty-bin` method.